Patch from FreeBSD - https://svnweb.freebsd.org/ports/head/databases/mariadb-connector-odbc/files/patch-CMakeLists.txt?view=log&pathrev=558761

Fix libcrypto & libssl linking issue:
...
Error relocating /usr/lib/libmaodbc.so: SSL_free: symbol not found
Error relocating /usr/lib/libmaodbc.so: compress: symbol not found
...

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -229,7 +229,7 @@ IF(WIN32)
   ENDIF()
 ELSE()
   SEARCH_LIBRARY(LIB_MATH floor m)
-  SET(PLATFORM_DEPENDENCIES ${LIB_MATH})
+  SET(PLATFORM_DEPENDENCIES ${PLATFORM_DEPENDENCIES} ${LIB_MATH})
   SET (MARIADB_ODBC_SOURCES ${MARIADB_ODBC_SOURCES}
                         ma_platform_posix.c
                         ma_conv_charset.c)
